configs/nvm/sweep_hybrid.py

```diff
@@ -78,9 +78,9 @@
 parser.add_option("--nvm-perc", type="int", default=100,
                   help = "Percentage of NVM commands")
 
-parser.add_option("--mode", type="choice", default="HYBRID",
-                  choices=hybrid_generators.keys(),
-                  help = "Hybrid: Random DRAM + NVM traffic")
+#parser.add_option("--mode", type="choice", default="HYBRID",
```

Comment:
Why this change?

Reply:
This file can be ignored. I was just reading it and trying to run it, but it was failing. So, I had to made this change to run.